### ISSUE TITLE
Docblockr wrap lines error

### ISSUE THREAD

**Issue by ilanbiala** [nikhilkalige/docblockr#16]
Uncaught TypeError: Cannot read property '1' of null 
/Users/ilanb/.atom/packages/docblockr/lib/docblockr-worker.js:561

**Comment by ilanbiala** [comment]
@NikhilKalige is this going to be fixed?

**Comment by malko** [comment]
With latest atom and docblocker version seems already resolved. Can you confirm there's no more problem about this plz ?

**Comment by ilanbiala** [comment]
still an issue for me.

**Comment by NikhilKalige** [comment]
```
/**
 * Keeping packages up to date helps keep things speedy and breakage to a minimum. Upgrade often! Keeping packages up to date helps keep things speedy and breakage to a minimum. Upgrade often!
 * Keeping packages up to date helps keep things speedy and breakage to a minimum. Upgrade often! Keeping packages up to date helps keep things speedy and breakage to a minimum. Upgrade often!
 * Keeping packages up to date helps keep things speedy and breakage to a minimum. Upgrade often! Keeping packages up to date helps keep things speedy and breakage to a minimum. Upgrade often!
 */
```

`Docblockr: Wrap Lines`

```
/**
 * Keeping packages up to date helps keep things speedy and breakage to a
 * minimum. Upgrade often! Keeping packages up to date helps keep things
 * speedy and breakage to a minimum. Upgrade often! Keeping packages up to
 * date helps keep things speedy and breakage to a minimum. Upgrade often!
 * Keeping packages up to date helps keep things speedy and breakage to a
 * minimum. Upgrade often! Keeping packages up to date helps keep things
 * speedy and breakage to a minimum. Upgrade often! Keeping packages up to
 * date helps keep things speedy and breakage to a minimum. Upgrade often!
 */
```
Can you put up an example?

**Comment by ilanbiala** [comment]
I get an error when I try to wrap a multiline decorated comment. That case isn't handled.

**Comment by malko** [comment]
Ok confirmed here, started to work on this one not so trivial i'll have another look later.
For now i spotted some exp which need rework to get this working for multiline:

in docblockr-worker.js:
line 556: 
```
_col = _text.search(/^\s*\*(\/)/);
// should become
_col = _text.search(/^\s*\*+(\/)/);
```

line 577 is not made to work with multiline comments either.

finally even getting the two previously mentioned ok it still need to reduce first and ending decoration line length which i had no time to get a look in.

will check again another time.

**Comment by jordwalke** [comment]
This seems like a pretty big issue - still experiencing it.

**Comment by MoritzKn** [comment]
Fixed by #222. Released with version `v0.8.7`.
